package.json

```diff
@@ -53,9 +53,10 @@
     "watch:js": "nodemon -w assets/js -x 'npm-run-all -s lint:client uglify'",
     "browserify": "browserify ./assets/js/index.js > ./public/js/bundle.js",
     "create:public": "mkdir -p ./public/css ./public/images ./public/js",
-    "postinstall": "npm-run-all create:public sass copy:images uglify",
+    "postinstall": "npm-run-all create:public sass copy:images uglify hooks",
     "snyk-protect": "snyk protect",
-    "prepublish": "npm run snyk-protect"
+    "prepublish": "npm run snyk-protect",
+    "hooks": "cd .git/hooks/ ; ln -fs ../../hooks/* ./"
```

Comment:
I think we need to guard against the case in which `.git/` doesn't exist.

Reply:
Fixed - `.git/hooks` created before linking